packages/devkit/src/generators/plugin-migrations/executor-to-plugin-migrator.ts

```diff
@@ -29,8 +29,9 @@ import type { ConfigurationResult } from 'nx/src/project-graph/utils/project-con
 type PluginOptionsBuilder<T> = (targetName: string) => T;
 type PostTargetTransformer = (
   targetConfiguration: TargetConfiguration,
-  tree?: Tree,
-  projectDetails?: { projectName: string; root: string }
+  tree: Tree,
+  projectDetails: { projectName: string; root: string },
+  inferredTargetConfiguration: TargetConfiguration
```

Comment:
With these not being optional now, will it break any existing migrations?

Reply:
It doesn't. The function is always invoked with those params provided. They are not optional. It's up to the consumer to use them or not. You can see the build and tests are still passing.